server/portal/apps/projects/management/commands/projects_id.py

```diff
@@ -1,96 +1,72 @@
 """Management command."""
 
 from django.conf import settings
+from django.db import transaction
+from django.core.exceptions import ObjectDoesNotExist
 from django.core.management.base import BaseCommand
 from portal.libs.agave.utils import service_account
-from portal.libs.agave.models.files import BaseFile
 from portal.apps.projects.models.base import ProjectId, Project
+from portal.libs.agave.operations import iterate_listing
 
 
 class Command(BaseCommand):
-    """Command class."""
-
     help = (
-        'Manage projects next Id. By default this command will print '
+        'Manage projects latest project i. By default this command will print '
         'the current latest project id, the last project id used in '
-        'storage systems and the last project id used in folders created '
-        'in corral.'
+        'storage systems and the last project id used in folders created.'
     )
 
-    def _get_latest_project_storage(self):  # pylint:disable=no-self-use
-        """Get latest agave project storage."""
+    @staticmethod
+    def get_latest_project_storage(ignore_above_value=None):
+        """Get latest agave project storage.
+
+        :param ignore_above_value: If provided, then ignore projects ids that are greater than or equal to this value.
+        """
         offset = 0
-        limit = 100
-        prjs = Project.listing(
-            service_account(),
-            offset=offset,
-            limit=limit
-        )
-        done = False
+        limit = 1000
         latest = -1
-        while not done:
-            prj_id = -1
+        while True:
+            prjs = Project.listing(
+                service_account(),
+                offset=offset,
+                limit=limit
+            )
+            prjs = [p for p in prjs]
             for prj in prjs:
                 prj_id = prj.storage.id.replace(
-                    settings._PORTAL_DATA_DEPOT_PROJECTS_SYSTEM_PREFIX,
+                    settings.PORTAL_PROJECTS_NAME_PREFIX,
```

Comment:
I see why you made this change, but this after looking into it, I had a thought:

- In [project_id_to_system_id](https://github.com/TACC/Core-Portal/blob/main/server/portal/apps/projects/utils.py#L41), Project systems are instantiated with `PORTAL_PROJECTS_NAME_PREFIX`.
- However, in [SystemsListView](https://github.com/TACC/Core-Portal/blob/main/server/portal/apps/accounts/api/views/systems.py#L39), systems are filtered out by `PORTAL_DATA_DEPOT_PROJECTS_SYSTEM_PREFIX`

Is the second point a bug? I think in most cases, we may have been setting 
`_PORTAL_PROJECTS_NAME_PREFIX` to equal `_PORTAL_DATA_DEPOT_PROJECTS_SYSTEM_PREFIX`, so it works out. But it looks like `_PORTAL_DATA_DEPOT_PROJECTS_SYSTEM_PREFIX` isn't really used. Should we deprecate that setting?

Reply:
💯  Yes!  I saw this too and first considered doing it here but then thought it would complicate things.  I meant to create a ticket but totally forgot.  Creating a ticket ✍️ 🎗️

Reply:
[Task FP-995](https://jira.tacc.utexas.edu/browse/FP-995) created and added to next sprint.